lib/draft/schema.json

```diff
@@ -958,15 +960,7 @@
         "bridgeInterfaces": {
           "type": "array",
           "items": {
-            "enum": [
-              "modbus",
-              "m-bus",
-              "can bus",
-              "rs-485",
-              "sdi-12",
-              "analog",
-              "dali"
-            ]
+            "enum": ["modbus", "m-bus", "can bus", "rs-485", "sdi-12", "analog"]
```

Comment:
please leave them in a list form for order, and `dali` is missing

Reply:
Fixed the mistake.

NB: also added downlink codecs.